source/ui.py

```diff
@@ -55,8 +55,9 @@ def browseableMessage(message,title=None,isHtml=False):
 	if not title:
 		# Translators: The title for the dialog used to present general NVDA messages in browse mode.
 		title = _("NVDA Message")
-	isHtmlArgument = "true" if isHtml else "false"
-	dialogString = u"{isHtml};{title};{message}".format( isHtml = isHtmlArgument , title=title , message=message ) 
+	if not isHtml:
+		message = f"<pre>{message.replace('<', '&lt;')}</pre>"

```

Comment:
Please use something like html.escape() to ensure that the text is totally safe to include within html. &lt; is not enough.

Reply:
Thanks for this suggestion.

According to the [French MDN page](https://developer.mozilla.org/fr/docs/Web/HTML/Element/pre):

> L'élément HTML <pre> représente du texte préformaté, généralement écrit avec une police à chasse fixe. Le texte est affiché tel quel, les espaces utilisés dans le document HTML seront retranscrits.
> Note : Il est nécessaire d'échapper les caractères '<' en '&lt;' afin de s'assurer que le code écrit entre les éléments ne soit pas interprété de façon involontaire.

Roughly translated (Google Translate) as follows:

> The HTML <pre> element represents preformatted text, typically written in a fixed-width font. The text is displayed as is, the spaces used in the HTML document will be transcribed.
> Note: It is necessary to escape the characters '<' in '<' to ensure that the code written between the elements is not unintentionally interpreted.

Thus escaping < character seems to be enough (confirmed by the tests).

However, the [English MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre) does not indicate this note.

Thus, for more safety, I will use `html.escape`anyway, as suggested by you.